src/comp_loinc/loinc_builder_steps.py

```diff
@@ -786,13 +786,12 @@ def save_to_owl(
         ] = None,
     ):
         """Save the current module to an OWL file."""
-        logger.info(f"Starting save-owl")
+        logger.info(f"save-owl: Starting")
```

Comment:
Updates to `save_to_owl()` logging.

Mainly, Removed the logging for how long it actually takes to write to file, as this was almost nominal. Now times the start/end for the whole function, which is basically taken up entirely by `owl_dumper.to_ontology_document()`.